retriever/.bumpversion.cfg

```diff
@@ -0,0 +1,6 @@
+[bumpversion]
+current_version = 0.2.0
```

Comment:
I guess we should have to change this to 0.2.1

`current_version = 0.2.1`

Reply:
I agree, accepted 0.2.1